### DISCUSSION TITLE
Another Vercel account is using this domain

### DISCUSSION THREAD

**Discussion by empato-limited** [vercel/community#2940] [Help] [answered]
### Summary

Another Vercel account is using this domain. Please set the following TXT record on _vercel.teeps.app to use blog.teeps.app in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.

I've added the TXT required, the error persists. 

<img width="936" alt="Screenshot 2023-06-21 at 12 52 16" src="https://github.com/vercel/community/assets/118970534/f6ec8df8-9483-4ee3-8b8f-465cc56ab684">

<img width="1283" alt="Screenshot 2023-06-21 at 12 53 12" src="https://github.com/vercel/community/assets/118970534/1a811316-121f-47b2-8e86-d62b9dae62c2">

### Steps to Reproduce

I've added the TXT required, the error persists. 

<img width="936" alt="Screenshot 2023-06-21 at 12 52 16" src="https://github.com/vercel/community/assets/118970534/f6ec8df8-9483-4ee3-8b8f-465cc56ab684">

<img width="1283" alt="Screenshot 2023-06-21 at 12 53 12" src="https://github.com/vercel/community/assets/118970534/1a811316-121f-47b2-8e86-d62b9dae62c2">

**Comment by amyegan** [comment, marked as answer]
Hi @empato-limited! It looks like you're trying to add the domain from a third party registrar. The `TXT` record should be added with your domain registrar alongside the `A` and `CNAME` records you used to connect the custom domain to Vercel.

**Reply by empato-limited** [reply]
Thank you.
